Yoast/javascript
# Add @yoast/search-metadata-previews package

Fixes https://github.com/Yoast/javascript/issues/31

Moved the following to a new package (`@yaost/search-metadata-previews`)
- `yoast-components/composites/SnippetEditor`
- `yoast-components/composites/SnippetPreview`

There still are some references to yoast-components, which will trigger an ECYCLE dependency cycle warning. I think we can ignore these until we've moved all code from `yoast-components/composites/plugin/shared` to another package or find another solution.

## Test instructions
- run `yarn test`. This should succeed. 
- run `yarn lint`. This should succeed.
